frontend/src/app/AdminNavBar/AdminNavBar.tsx

```diff
@@ -118,25 +120,30 @@ export const AdminNavBar = ({ isMenuOpen }: AdminNavBarProps): JSX.Element => {
 
   // Emergency contact modal appears after the rollout announcement modal
   useEffect(() => {
-    if (!hasSeenContactModal && user && !user?.contact && hasSeenAnnouncement) {
+    if (
+      hasSeenContactModal === false &&
+      user &&
+      !user.contact &&
+      hasSeenAnnouncement === true
```

Comment:
(for my learning!) - do we use `hasSeenAnnouncement === true` instead of just `hasSeenAnnouncement` for readability?

Reply:
no, we could probably use `hasSeenAnnouncement` shorthand, oops hehe